pkg/providers/aws/cluster_network_provider.go

```diff
@@ -890,7 +890,7 @@ func (n *NetworkProvider) reconcileStandaloneRouteTableTags(ctx context.Context,
 	for _, routeTable := range routeTables {
 		routeTableTags := ec2TagsToGeneric(routeTable.Tags)
 		if !tagsContains(routeTableTags, tagDisplayName, DefaultRHMIVpcRouteTableNameTagValue) ||
-			!tagsContains(routeTableTags, aws.StringValue(croOwnerTag.Key), aws.StringValue(croOwnerTag.Value)) {
+			!tagsContains(routeTableTags, *aws.String(croOwnerTag.key), *aws.String(croOwnerTag.value)) {
```

Comment:
I dont think this looks right, can we not use `aws.StringValue`?

Reply:
yep. it's a generic tag now so don't need those, will remove